### ISSUE TITLE
exp.show_in_notebook(show_table=True) renders poorly with a regression explanation

### ISSUE THREAD

**Issue by ianozsvald** [marcotcr/lime#88]
I'm not sure what I should see but the rendered output doesn't look right. The screenshot is taken from my rendered public Notebook:

![image](https://user-images.githubusercontent.com/273210/28747833-243a939c-74a0-11e7-889c-571ad694bafd.png)

I'm flagging this as I believe the regression support in LIME is young, so maybe this rendering isn't as expected?

The middle display (Negative/Positive) only shows 5 items and feels cramped. The explanation is generated using `exp = explainer.explain_instance(example_to_explain, est.predict, num_features=10)`.

The Notebook is available here: https://github.com/ianozsvald/data_science_delivered/blob/master/ml_explain_regression_prediction.ipynb

`watermark` reports the following versions (it might be worth setting a version string for `lime` too):

```
CPython 3.6.2
IPython 6.1.0

numpy 1.13.1
matplotlib 2.0.2
sklearn 0.18.2
eli5 0.6.1
seaborn 0.8.0
lime n
yellowbrick 0.4.2

compiler   : GCC 4.8.2 20140120 (Red Hat 4.8.2-15)
system     : Linux
release    : 4.9.8-040908-generic
machine    : x86_64
processor  : x86_64
CPU cores  : 8
interpreter: 64bit
Git hash   : d1099144f8b748e8e5aa43676f24bc8cba7204c9
```

**Comment by marcotcr** [comment]
What version of LIME do you have? I ran the notebook and this is what I get:
![image](https://user-images.githubusercontent.com/698010/29187428-909302d6-7dc4-11e7-854c-9b7d23b87080.png)

Try installing the newest version and clearing your cache (shift + reload should work on the notebook).

**Comment by ianozsvald** [comment]
Hi Marco. I'm using `master` pulled two weeks back. Which Jupyter are you using? I have
```
jupyter --version
4.3.0
```

Are there any plugins that are responsible for the rendering that I should version check?

**Comment by marcotcr** [comment]
I have 4.3.2
I don't know what this could be, sorry!

**Comment by nasir-bhanpuri** [comment]
I was having the same issue with firefox, but seems to render better with chrome.

**Comment by Claudia1806** [comment]
Had the same problem -> using chrome worked for me too

**Comment by marcotcr** [comment]
I think #296 fixes this, let me know if it does not.
